pom.xml

```diff
@@ -97,6 +97,12 @@
 
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>credentials</artifactId>
```

Comment:
Why adding this dependency?

Reply:
Error class `SecretBytes` NotFound thrown on my local env. I fixed by adding the dependency. Any other way to fix it?

Reply:
Not sure if `SecretBytes` is required by this plugin or azure-credentials plugin. If the latter case, maybe we should add it to the azure-credentials plugin?

Reply:
Your are right, we can add it to azure-credential later. I revoked the change. The exception was warning only. I can ignore it..

Reply:
Have to add it back. Otherwise it breaks CI.